sktwocnt/prog/output.f90

```diff
@@ -17,8 +18,19 @@ subroutine write_sktables(skham, skover)
     !> Hamiltonian and overlap matrix
     real(dp), intent(in) :: skham(:,:), skover(:,:)
 
-    call write_sktable_("at1-at2.ham.dat", skham)
-    call write_sktable_("at1-at2.over.dat", skover)
+    if (size(skham, dim=2) > 0) then
```

Comment:
Not sure if I've followed the logic fully, but what happens if non-lead processes get here?

Reply:
They don't/shouldn't call this routine. If they would, all processes will write exactly the same file, with the same name and content (don't know how the filesystem handles this), as all the data is present on all ranks.